config/runtime/fstests.jinja2

```diff
@@ -1,32 +1,60 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
- 
+
 {%- extends 'base/python.jinja2' %}
- 
+
 {%- block python_imports %}
 {{ super() }}
 import subprocess
 {%- endblock %}
- 
+
 {%- block python_globals %}
 {{ super() }}
 REVISION = {{ revision }}
 {% endblock %}
```

Comment:
I don't think this is used either, but it was already there from the previous commits.  Maybe something to clean up at some point.

Reply:
updated